recipes/awslambdaric/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "awslambdaric" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/awslambdaric-{{ version }}.tar.gz
+  sha256: dad646f566aa7ec9b7179f16ca6741a2bea148abec6ed5947f86d00607e0a9a2
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
```

The linux version of this library uses the native aws-lambda-cpp SDK. We need the conda-forge compiler toolchain during build time. These requirements take care of that.
```suggestion
   build:
     - {{ compiler('c') }}  # [linux]
     - {{ compiler('cxx') }}  # [linux]
     - automake  # [linux]
     - autoconf  # [linux]
     - cmake  # [linux]
     - libtool  # [linux]
     - make  # [linux]
     - pkg-config  # [linux]
  host:
```